src/fe-common/core/fe-windows.c

```diff
@@ -438,35 +468,34 @@ WINDOW_REC *window_find_item(SERVER_REC *server, const char *name)
 	return window_item_window(item);
 }
 
+static GSequenceIter *windows_seq_refnum_search_right(int refnum)
```

Comment:
Please add a small comment about the different implementation of the `left` vs `right` cases for our future selves and/or other devs.